src/operations/delete.ts

```diff
@@ -80,6 +94,10 @@ export class DeleteOperation extends CommandOperation<Document> {
       command.let = options.let;
     }
 
+    if (options.comment) {
```

Comment:
We were applying the `comment` option in the delete statement portion of the command, and not at the top level.  See https://docs.mongodb.com/manual/reference/command/delete/

Reply:
Does it work in both places? the linked example only shows it in the top level document

Reply:
It's still in the DeleteStatement type, should it just be removed from there? I see its public but if it never worked then maybe it was bug that it was defined there